source/server/code/views/choose_ltf.erb

```diff
@@ -6,19 +6,32 @@ $(() => {
 
   $('.type').html(cd.urlParam('type'));
 
-  $('button.next').click(() => {
-    const name = encodeURIComponent(cd.selectedDisplayName);
-    cd.goto(`/creator/confirm?${cd.urlParams()}&language_name=${name}`);
+  $('button.ok').click(() => {
+      const name = encodeURIComponent(cd.selectedDisplayName);
+      $.ajax({
+          type: 'POST',
+          url: `/creator/confirm?${cd.urlParams()}&language_name=${name}`,
+          dataType: 'json',
+          async: false,
```

Comment:
$.ajax call with async: false generates a warning on the browser:
`[Deprecation] Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check https://xhr.spec.whatwg.org/.`
To reproduce, I commented the `$('.home-icon').trigger("click");` block and clicked the button. A new tab is opened, and on the original tab, on the console, the warning is shown.

Reply:
Good find. I didn't know about that.

Reply:
And to be clear - I assumed that `async:false` was needed but its obviously better if it works with `async:true`